app/assets/stylesheets/components/_accessible-autocomplete.scss

```diff
@@ -0,0 +1,35 @@
+@import url(asset-path("accessible-autocomplete/dist/accessible-autocomplete.min.css")); /* stylelint-disable-line */
+
+.app-c-accessible-autocomplete {
+  .autocomplete__input {
+    z-index: 1;
+  }
+
+  .autocomplete__dropdown-arrow-down {
+    z-index: 0;
+  }
+
+  .autocomplete__option {
```

Comment:
I think some additional styles might be needed to fix a problem with the first, blank option in the autocomplete, which is shown below as the dark blue highlighted strip.

![Screenshot 2023-04-17 at 10 30 46](https://user-images.githubusercontent.com/861310/232444812-9dc08ca9-cd91-4a3b-8587-a9c4d1048fbf.png)

As you can see it doesn't look like a selectable option, but it  allows users to clear the selection in the autocomplete by clicking, rather than having to delete the text manually if required. I remember this from the first implementation, it should be fixable with something like `min-height: 1em;`, which would result in the following:

![Screenshot 2023-04-17 at 10 34 07](https://user-images.githubusercontent.com/861310/232445454-ad8d8535-9006-4807-9e41-f5008a4201ea.png)